src/Umbraco.Web.UI.Client/src/common/directives/components/tabs/umbtabsnav.directive.js

```diff
@@ -145,7 +145,6 @@ Use this directive to render a tabs navigation.
             vm.needTray = false;
             vm.showTray = false;
             vm.overflowingSections = 0;
-            vm.maxTabs = 7;
```

Comment:
@rasmusjp you have remove this variable `vm.maxTabs`. Do we still need it in the view?
https://github.com/umbraco/Umbraco-CMS/blob/v8/dev/src/Umbraco.Web.UI.Client/src/views/components/tabs/umb-tabs-nav.html#L2

Reply:
We do, it's still being set in the calculateWidth function